.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "editor.formatOnSave": true,
+    "[toml]": {
+        "editor.formatOnSave": false
+    },
+    "rust-analyzer.check.allTargets": false,
+    "rust-analyzer.check.noDefaultFeatures": true,
+    "rust-analyzer.cargo.noDefaultFeatures": true,
+    // "rust-analyzer.cargo.target": "thumbv7em-none-eabi",
+    "rust-analyzer.cargo.target": "thumbv8m.main-none-eabihf",
+    "rust-analyzer.cargo.features": [
+        "stm32h743zi"
+    ],
+    "rust-analyzer.linkedProjects": [
+        "client/embedded-app/Cargo.toml",
+        // "server/embedded-app/Cargo.toml",
+    ],
+}
```

Comment:
Should this file be necessary given the VSC workspace file? I find that I don't need it.